karma.conf.js

```diff
@@ -0,0 +1,29 @@
+module.exports = function (config) {
+  const configuration =
+    {
+      files: [
+        'test/bundle.js'
+      ],
+
+      frameworks: [
+        'mocha'
+      ],
+
+      customLaunchers: {
+        Chrome_travis_ci: {
+          base: 'Chrome',
+          flags: ['--no-sandbox']
+        }
+      },
+
+      browsers: [
+        'Firefox',
+        'Chrome',
+      ],
+    }
+
+  if (process.env.TRAVIS)
+    configuration.browsers = ['Chrome_travis_ci', 'Firefox'];
+
+  config.set(configuration);
+}
```

Comment:
No new line EOF. This is needed because of Git diffs. You should set this as a standard so you need not to think about it.